content/java/_index.md

```diff
@@ -0,0 +1,130 @@
+---
+title: "WorldWind Java"
+date: 2017-07-10T23:14:25-04:00
+draft: false
+mainpage: false
+projectpage: false
+projectslug: "java"
+projectname: "WorldWind Java"
+listdescription: ""
+aliases:
+    - /java/get-started/
+---
+
+## Get Started
+
+Built on Java and OpenGL, WorldWind Java is a cross-platform virtual globe in use by thousands of organizations around the world. With support for an array of different data formats WorldWind Java provides a highly configurable visual representation of a globe for you geographic needs.
+
+### Run an Application Using WorldWind
+
+1. Download the latest release from the WorldWind [GitHub Releases](https://github.com/NASAWorldWind/WorldWindJava/releases).
+
+2. Extract the WorldWind SDK to a folder anywhere on your hard drive. The following steps refer to the extracted folder as WorldWind.
+
+3. Run a WorldWind Demo by opening the WorldWind/README.txt and following the instructions under "Running a Basic Demo Application".
+
+4. Create an app by reviewing the programming examples in WorldWind/src/gov/nasa/worldwindx/examples. Start with SimplestPossibleExample and ApplicationTemplate, then refer to the [Examples](/java/examples/) page for more advanced topics. Here are examples of running your app from the command-line, assuming your app's JAR file is MyApp.jar, and your main class is worldwinddemo.MyBasicDemo.
+
+On Mac OS X and Linux
+
+```
+java -Xmx1024m -cp MyApp.jar:worldwind.jar:worldwindx.jar:jogl-all.jar:gluegen-rt.jar:gdal.jar worldwinddemo.MyBasicDemo
+```
+
+On Windows
+
+```
+java -Xmx1024m -Dsun.java2d.noddraw=true -cp MyApp.jar;worldwind.jar;worldwindx.jar;jogl-all.jar;gluegen-rt.jar;gdal.jar worldwinddemo.MyBasicDemo
+```
+
+Note: All *.dll, *.jnilib, and *.so files in the WorldWind folder must be in the same folder as MyApp.jar.
```

Comment:
On line 40, the font has switched to italics for the rest of the content (in the GitHub diff viewer).